docs/source/conf.py

```diff
@@ -62,7 +62,7 @@
     'sphinx.ext.mathjax',
     'sphinx.ext.napoleon',
     'sphinx_copybutton',
-    'sphinx_autodoc_toolbox.collapse',
```

Comment:
I use `sphinx_autodoc_toolbox.collapse` because I want to collapse details of the class members like [here](https://docs.abqpy.com/en/latest/reference/mdb.html) to make the API docs not too long, I developed the extension in [haiiliin/sphinx-autodoc-toolbox](https://github.com/haiiliin/sphinx-autodoc-toolbox/)

Reply:
oh, ok!
I couldn't find the extension, so I thought it was written with the wrong name